themes/codefor-theme/layouts/partials/map-overview.html

```diff
@@ -52,14 +52,23 @@
       map.addControl(L.control.zoom({ position: 'topright' }));
 
       // basemap
+      // NOTE: Do not use Mapbox with time limited token!
```

Comment:
No need to add a comment here. The reason for switching can go into the commit message :)

Reply:
Comments removed